envwarden

```diff
@@ -25,6 +25,15 @@ usage()
     echo "You can use ~/.envwarden to store your credentials (just email, or email:password)"
 }
 
+check_version()
+{
+    printf '%s\n%s' $2 $3 | sort -C -V
```

Comment:
I tested it on an Ubuntu box, which produced `jq-1.5-1-a5b5cbe` and then the comparison produces a false negative... `sort -V` is a great find, but I'm not sure it really works, at least not without heavily normalizing the `jq --version` output, which I'm not 100% sure about its format... :(